lib/ecl/ecl_file_view.cpp

```diff
@@ -33,7 +35,7 @@
 struct ecl_file_view_struct {
   vector_type       * kw_list;      /* This is a vector of ecl_file_kw instances corresponding to the content of the file. */
   hash_type         * kw_index;     /* A hash table with integer vectors of indices - see comment below. */
-  stringlist_type   * distinct_kw;  /* A stringlist of the keywords occuring in the file - each string occurs ONLY ONCE. */
+  std::vector<std::string> * distinct_kw;  /* A list of the keywords occuring in the file - each string occurs ONLY ONCE. */
```

Comment:
I was wondering, do we really need to make this dynamically allocated? This can easily become a class member and there will be no need for `new` and `delete`

Reply:
Thank you for the review. Yes I would also have preferred the `distinct_kw` to be a class member, but that would require that the parent - i.e. `ecl_file_view` was handled as a proper class, and not just a C struct as is yet the case - and that would be more invasive change.